### PR TITLE
feat(jest): 42% coverage for BlobManip

### PR DESCRIPTION
**Pull request by drepram**
**What this PR does** 📖

Self-explanatory. Commented out code at the last part of the code is due to possibly JSDOM being weird and not functioning well with FileStreams

**Which issue(s) this PR fixes** 🔨

AP-2257

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Review by WanderingHogan — CHANGES_REQUESTED** [review]
If blob doesn't exist in jsdom, is there a polyfill we can use, or can we mock it? As it stands this will always just fail out and never run this test

**Comment by stavares843** [comment]
in case if we can't do what hogan mentioned above, please just ditch the commented out code

File: utilities/BlobManip.test.ts [hunk 1]
```diff
@@ -49,3 +49,34 @@ describe('convert blob to base 64 failing', () => {
...
+    //   })
+    // })
+    // afterAll(() => {
+    //   Object.defineProperty(global, 'ReadableStream', {
+    //     writable: true,
+    //     value: original,
+    //   })
+    // })
+    try {
+      const jpegBlob = await new Blob(['testing'], { type: 'image/jpeg' })
```
Comment (josephmcg) [inline utilities/BlobManip.test.ts:76]: this would be a text file right? not really a jpeg

Comment (drepram) [reply utilities/BlobManip.test.ts]: ah yes, correct. please check again

**Comment by stavares843** [comment]
@drepram 


in case if we can't do what hogan mentioned above, please just ditch the commented-out code

**Comment by WanderingHogan** [comment]
There are a couple things here - 

We shouldn't worry about commented out code when it's stuff that's in flight and might help figure things out later. We should remove dead code, but I don't think this is the case here.

As-is, i don't think we should check this in, because it will never actually get to the test unless we can mock it (which i couldn't figure out how to do in the time I had to look at this)

Screen Shot 2022-09-01 at 12 14 31 PM
Screen Shot 2022-09-01 at 12 14 35 PM

**Comment by stavares843** [comment]
will close this then, if you figure out a solution we can reopen @drepram

**Pull request closed without merging**